src/Microsoft.AspNetCore.Testing/ExceptionAssertions.cs

```diff
@@ -85,9 +83,10 @@ public static TException Throws<TException>(Func<object> testCode, string except
         /// <param name="exceptionMessage">The exception message to verify</param>
         /// <returns>The exception that was thrown, when successful</returns>
         /// <exception>Thrown when an exception was not thrown, or when an exception of the incorrect type is thrown</exception>
-        public static ArgumentException ThrowsArgument(Action testCode, string paramName, string exceptionMessage)
+        public static TException ThrowsArgument<TException>(Action testCode, string paramName, string exceptionMessage)
```

Comment:
Why is this now generic? I can understand generic helper methods but this will break many repos due to reuse within this class.